keystorage/yubihsm2/src/main/java/tech/pegasys/signers/yubihsm/exceptions/YubiHsmException.java

```diff
@@ -10,30 +10,29 @@
  * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
  * specific language governing permissions and limitations under the License.
  */
-package tech.pegasys.signers.yubihsm2;
+package tech.pegasys.signers.yubihsm.exceptions;
+
+import tech.pegasys.signers.yubihsm.model.Errors;
 
 public class YubiHsmException extends RuntimeException {
-  public YubiHsmException() {
-    super();
-  }
+  Errors yhError;
```

Comment:
probably deserves a full name, either ditch the yh, or name it full to yubiHsmError